Makefile

```diff
@@ -37,6 +37,7 @@ prepare-dev:
 	fi
 	python3 -m pip install pipenv
 	python3 -m venv $(VENV_NAME)
+	$(PYTHON) -m pip install enquiries
```

Comment:
wondering why do we do differently here between `python3` and `$(PYTHON)`? 🤔  I thought we can unify it no?

Reply:
@sleepylee Because we run the script from `$(PYTHON) ./scripts/setup.py` then we have to install the package to that `$(PYTHON)` 🤔 
I tried to `python3 -m pip install enquiries` but we got an error (package not found) when running the script